edge-util/docker/linux/amd64/Dockerfile

```diff
@@ -1,12 +1,13 @@
-ARG num_procs=4
-
+# docker file for azureiotedge/azureiotedge-runtime-base:1.1-linux-amd64
 FROM alpine:3.7
 
+ARG num_procs=4
```

Comment:
Previously this argument doesn't work, and therefore make command will run number of processes same as number of cores.  However this may run out of memory and cause compiler error.  Therefore I updated OneNote to pass 1 for this arg if compiler error happens.

OneNote: https://microsoft.sharepoint.com/teams/Azure_IoT/_layouts/OneNote.aspx?id=%2Fteams%2FAzure_IoT%2FShared%20Documents%2FEngineering%2FIoTHubNotes&wd=target%28Azure%20IoT%20Edge%2FWiki.one%7C70253E27-7EC5-43AD-89E4-C9BA3943D8CF%2FLinux%20x64%7C77BBFAC0-DDF1-45C7-B4A6-229692CFD066%2F%29
onenote:https://microsoft.sharepoint.com/teams/Azure_IoT/Shared%20Documents/Engineering/IoTHubNotes/Azure%20IoT%20Edge/Wiki.one#Linux%20x64&section-id={70253E27-7EC5-43AD-89E4-C9BA3943D8CF}&page-id={77BBFAC0-DDF1-45C7-B4A6-229692CFD066}&end